rseqc-hg38-gencode23/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:14.04
+
+MAINTAINER John Vivian, jtvivian@gmail.com
+
+RUN apt-get update && apt-get install -y \
+            build-essential \
+            python-dev \
+            R-base \
+            samtools \
+            python-pip \
+            python-numpy
+
+RUN pip install cython
+
+# Install and setup RSeQC
+COPY RSeQC-2.6.3.tar.gz /opt/
```

Comment:
Why is this tarball part of the source tree? Is there no way to download it from an external source?